repo_policy_compliance/github_client.py

```diff
@@ -148,6 +148,10 @@ def get_collaborator_permission(
     Returns:
         The collaborator permission.
     """
-    return cast(
-        Literal["admin", "write", "read", "none"], repository.get_collaborator_permission(username)
-    )
+    user_permission = repository.get_collaborator_permission(username)
+    if user_permission not in ("admin", "write", "read", "none"):
+        raise GithubClientError(
```

Comment:
you probably will get a lint failure since this exception is not documented in the docstring 🙂